data/World/Forest Temple.json

```diff
@@ -10,6 +10,13 @@
                 has_bombchus or can_use(Dins_Fire) or (logic_forest_first_gs and (Bombs or
                     (can_jumpslash and can_take_damage)))",
             "Forest Temple GS Lobby": "can_use(Hookshot) or can_use(Boomerang)",
+            "Forest Temple Center Room Pot 1": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Center Room Pot 2": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Center Room Pot 3": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Center Room Pot 4": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Center Room Pot 5": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Center Room Pot 6": "(is_adult or can_child_attack or Nuts)",
+            "Forest Temple Lower Stalfos Pot": "(is_adult or can_child_attack or Nuts)",
```

Comment:
In my logic update PR I added a region for just dealing with the skulltula in the first hallway (long overdue).